src/rules/accessibility/tooltip-accessibility.js

```diff
@@ -0,0 +1,66 @@
+HTMLHint.addRule({
+    id: 'tooltip-accessibility',
+    description: 'element with binding "addTooltip" should have binding "addDescription"',
+    init: function (parser, reporter) {
+        var self = this;
+        var looseTooltips = [];
+        var isTooltip = function (event) {
+            var tagName = event.tagName;
+            var hasTooltipClass = HTMLHint.utils.isClassExsits(event.attrs, 'tooltip-help');
+            return tagName === 'span' && hasTooltipClass;
+        };
+        var isFocusableElement = function (event) {
+            var focusabaleTagNames = ['input', 'textare', 'select', 'a', 'button'];
+            return focusabaleTagNames.includes(event.tagName);
+        };
+        var getElementDescription = function (event) {
+            var bindings = HTMLHint.utils.getAttributeValue(event.attrs, "data-bind");
```

Comment:
make it more generic and expose in utils as function that gets an element and a binding name and returns a binding value (mihjt be empty string) or undefined if this binding doesn't exist